app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,19 @@
  *= require font-awesome
  */
 
+.homepage_headline {
+  vertical-align: text-top;
+  text-align: center;
+}
+
+.col-lg-10 {
+  display: block;
+  margin-top: 40px;
```

Comment:
Properties should be ordered display, margin-bottom, margin-left, margin-right, margin-top